test/verify/check-networking-firewall

```diff
@@ -177,6 +178,66 @@ class TestFirewall(MachineCase):
             b.click("caption button.btn-primary")
             b.wait_present("#cockpit_modal_dialog li input[data-id='empty']")
 
+    @skipImage("Custom ports were introduced in 191", "rhel-7-6-distropkg")
+    def testAddCustomServices(self):
+        b = self.browser
+
+        self.login_and_go("/network/firewall")
+
+        # click on the "Add Services" button
+        b.wait_present("caption button.btn-primary:enabled")
+
+        def open_dialog():
+            b.click("caption button.btn-primary")
+            b.wait_present("#add-services-dialog input[value='ports']")
+            b.click("#add-services-dialog input[value='ports']")
+            b.wait_present("#tcp-ports")
+            b.wait_visible("#tcp-ports")
+
+            time.sleep(1) # We need some time before /etc/services is parsed
```

Comment:
This needs to be fixed to wait on the effectively visible state change. Maybe the cleanest way would be to disable the button until services is read, or at least disable the "custom" radio button.

Reply:
I wanted not to disable anything as you still can create custom ports (although only with port numbers). But sure, I can disable radio button until file is read.